### DISCUSSION TITLE
Tooltip Information

### DISCUSSION THREAD

**Discussion by chale1342** [mattieserver/netbox-topology-views#568] [Q&A] [answered]
I am curious if there is a way to change what shows on the tool tip when hovering over a device? We would like to include comments/custom fields.

**Comment by dreng** [comment, marked as answer]
The information in the tooltip is not customizable. While it would be relatively easy to add a specific custom field, this would be a very poor implementation. On the other hand, implementing a well-designed feature for the user to add one or more custom fields can become a big task.

**Reply by chale1342** [reply]
That’s what I suspected. Thanks for the feedback.
